<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>The Veg-X exchange standard • VegX</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="The Veg-X exchange standard">
<meta property="og:description" content="VegX">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">VegX</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.6.12</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../articles/index.html">Articles</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="VegXStandard_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>The Veg-X exchange standard</h1>
                        <h4 data-toc-skip class="author">IAVS Ecoinformatics Working Group</h4>
            
            <h4 data-toc-skip class="date">2022-09-12</h4>
      
      
      <div class="hidden name"><code>VegXStandard.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="why-a-vegetation-data-exchange-standard">Why a vegetation data exchange standard?<a class="anchor" aria-label="anchor" href="#why-a-vegetation-data-exchange-standard"></a>
</h2>
<p>A primary technical impediment to large-scale sharing of vegetation data is the lack of a recognized international exchange standard for linking the panoply of tools and database implementations that exist among various organizations and individuals participating in vegetation research. In the absence of an exchange standard, the need for multiple, ad hoc mappings among databases and applications discourages merging of data and slows development of new analytical tools (Fig. 1a). By contrast, widespread use of a common exchange standard would avoid the need to repeatedly map data for synthetic projects by requiring only a single mapping between a given database or tool and the standard (Fig. 1b), thus facilitating data exchange and analysis. Application of an international exchange standard for vegetation data would form a critical part of the necessary infrastructure to allow these data to be combined for synthetic analysis at local and global scales.</p>
<div class="figure">
<img src="DataExchange.png" alt=""><p class="caption"><strong>Fig. 1</strong>. Schematic diagram showing (a) problem: multiple tools and databases and (b) solution: sharing tools through a common standard.</p>
</div>
</div>
<div class="section level2">
<h2 id="overview-of-the-veg-x-standard">Overview of the Veg-X standard<a class="anchor" aria-label="anchor" href="#overview-of-the-veg-x-standard"></a>
</h2>
<p>The <strong>Veg-X exchange standard</strong> for plot-based vegetation data (Wiser et al. 2011) is intended to be used to share and merge vegetation-plot data of different kinds. Veg-X allows for observations of vegetation at both individual plant and aggregated observation levels. It ensures that observations are fixed to physical sample plots at specific points in space and time, and makes a distinction between the entity of interest (e.g. an individual tree) and the observational act (i.e. a measurement). The standard supports repeated measurements of both individual organisms and plots, and enables the connection between the entity observed and the taxonomic concept associated with that observation to be maintained.</p>
<p>A goal in the creation of Veg-X was to have a schema that is relatively simple to read and use. To achieve this, highly nested structures were avoided and major vegetation data components were included (e.g. plot attributes, plot observations, organisms) as top-level elements that are referenced by each other through unique identifiers (e.g. a unique numerical ID) that allow the integrity of the original linkage to be captured. Although the main logical structure of vegetation data (i.e. the logical relationships between major data components) is fixed, alternative, user-defined ways of grouping observations are also allowed. As such, the standard can accommodate projects that are linked across time as well as longitudinal measures of plots or individuals to the extent that these are referenced in the original dataset through appropriate unique identifiers in those original sources.</p>
<p>The standard accommodates different data collection protocols by allowing specific aspects of data collection methods to be captured, such as whether plots were located subjectively or randomly, plot dimensions, definitions of cover-abundance scales, references to published measurement methods, etc. The standard also allows for the original units of measurement to be retained. All elements in the standard are clearly defined. This allows synonymous terms in source datasets to be mapped to a common set of concepts, thus overcoming the problems caused by inconsistent terminologies.</p>
<p>The <em>plotObservation</em> is the central Veg-X element, resulting from sampling a physical plot at a specific point in time, and can be related to one or more research projects (Fig. 2). The information about a sampled plot that is fixed over time (e.g. altitude, plot identifier or name, dimensions, aspect, slope, geology) and references to related plots (e.g. a parent plot) are stored in the separate element <em>plot</em>. By structuring the plot data in this way, repeat measures and nested plots can be accommodated in the standard.</p>
<p>Specific observations, either biotic or abiotic, are linked to the plot observation event. The standard allows storing observations of vegetation made at four different levels:</p>
<ul>
<li>
<strong>Individual organism</strong> level, such as tree diameter (element <em>individualOrganismObservation</em>)</li>
<li>
<strong>Aggregate organism</strong> level (i.e. characteristic of a group of individuals taken together), such as the percent cover of a taxon (element <em>aggregateOrganismObservation</em>).</li>
<li>
<strong>Stratum</strong> level, such as the basal area of trees of the emergent layer (element <em>stratumObservation</em>).</li>
<li>
<strong>Community</strong> level, such as the successional stage or species richness (element <em>communityObservation</em>).</li>
</ul>
<p>The standard maintains a clear distinction between the entity of interest (e.g. an individual organism, plot, or stratum) and the observation act (e.g. a measuring event applied to it). Together with unique identifiers that maintain the integrity of references between individual records within each component (e.g. between a plot and all the measuring events applied to it), the separation of components allows the standard to store multiple observations of the same entity (e.g. a plot or a tree). Analogously, a single observation event (e.g. a plot observation) may apply to multiple entities, thereby providing explicit grouping of entity observations. Each entity of interest (e.g. a tree) may have multiple observed properties (e.g. height, dbh) whose values are determined through measurement using a specific procedure or a method belonging to a particular protocol. Unlike individual organism observations, aggregate (i.e. collective) organism observations do not relate to a specific physical entity but provide estimates of the importance of a (abstract) taxonomic entity within the plot, such as through a cover estimate. Strata can be the subject of stratum observations (e.g. % of tree cover, tree height) and can be linked to aggregate organisms observations (Fig. 2).</p>
<p>The standard also maintains a distinction between identity of organisms (the taxon or taxon concept) and how these identities are applied to particular observations of organisms. This is done through three top-level elements:</p>
<ol style="list-style-type: decimal">
<li>Element <em>taxonConcept</em>: A specific taxonomic concept (i.e. a name–reference combination).</li>
<li>Element <em>organismIdentity</em>: The container for all information about the identity of an organism found in the plot observation (typically, this information includes a taxon name string possibly attached to taxonomic concept, but in some instances it may simply be the name of a morphospecies or a ‘field name’ used by the author of the data set). This element was called <em>taxonNameUsageConcept</em> in Wiser et al. (2011) and earlier versions of Veg-X.</li>
<li>Element <em>taxonDetermination</em>: An assertion, made by a party, linking one or more <em>taxonConcepts</em> to a <em>organismIdentity</em>, normally after re-examining the herbarium voucher of the specimen.</li>
</ol>
<p>All the organism observations referencing a given <em>organismIdentity</em> are affected by nomenclatural changes or determination events applied to it. This allows different determinations and taxonomic concepts to be associated with a vegetation entity so temporal changes in opinion regarding identification (i.e. “determination history”) can be recorded and both formal (i.e. taxon names) and informal (e.g. “field names”, “morphospecies”) names applied to a particular organism observation can be preserved. The fact that the <em>organismIdentity</em> is not nested within observations permits the same identity (i.e. name) to be reused within the scope of the individual dataset. Community determinations are handled in a similar way: <em>communityDetermination</em> elements allow a given plot observation to be related to one or multiple community concepts. Although the standard supports fully-specified taxonomic concepts, it does not require them. This is important as the full concept is unspecified and furthermore unrecoverable for most legacy data. On the other hand, because the schema can accommodate determination information (who did the identification, when, and with what reference), in theory it could be possible to recover concepts for many legacy datasets – in particular, tropical forest plots where such information is commonly preserved in the form of herbarium voucher specimens.</p>
<p>Veg-X is written as an XML schema, which is a definition of user-defined tags to structure textual information in order to create self-describing datasets. XML (Extensible Markup Language) is an open standard, and XML files are both machine and human-readable (they are stored in plain-text ASCII format). These characteristics help to ensure that data in this format will be accessible in the future. We made use of existing XML schema definitions, which we incorporated as modules of our schema. Specifically, we adapted parts of the <strong>Ecological Metadata Language</strong> (EML; <a href="https://knb.ecoinformatics.org/#tools/eml" class="external-link uri">https://knb.ecoinformatics.org/#tools/eml</a>; Jones et al. 2006) to define entities like projects, protocols, parties and methods. To specify taxon concepts, we used element names adopted from the <strong>Taxon Concept Transfer Schema</strong> (TCS; <a href="http://www.tdwg.org/standards/117/" class="external-link uri">http://www.tdwg.org/standards/117/</a>). TCS can be used to support taxon concept mappings (e.g. Franz &amp; Peet 2009).</p>
</div>
<div class="section level2">
<h2 id="the-vegx-r-package">The VegX R package<a class="anchor" aria-label="anchor" href="#the-vegx-r-package"></a>
</h2>
<p>A barrier to the use of a standard like Veg-X is its complexity, which is nevertheless required to accommodate the wide variety of schemes existing for vegetation plot data sources. The large data integration projects employ eco-informaticians to achieve this, but the tools developed are specific to these projects and cannot be readily picked up by others. To make the exchange schema of Veg-X usable by the wider community requires the development of informatics tools for mapping data from different input formats (e.g. relevé tables from different databases, forest inventory data or stem-mapped forest plots) into Veg-X, mechanisms to create unique identifiers to allow source datasets to be combined, and tools to export data from Veg-X to a range of formats that can serve as input to software packages for data analysis and visualisation.</p>
<p>In 2017, the <em>Ecoinformatics Working Group</em> of the <strong>International Association for Vegetation Science</strong> (IAVS; <a href="http://iavs.org/Working-Groups/Ecoinformatics.aspx" class="external-link uri">http://iavs.org/Working-Groups/Ecoinformatics.aspx</a> ) decided to develop an R package to promote the usage of the Veg-X standard. The R package is specifically intended to be used to:</p>
<ul>
<li>Integrate and harmonize vegetation-plot data from different sources with the aim of conducting new analyses.</li>
<li>Produce archivable XML documents for vegetation-plot that have no ‘home’ in existing vegetation plot databases such as those catalogued by the <strong>Global Index of Vegetation Plot Databases</strong> (GIVD; www.givd.info ) or as needed to support data availability requirements to support scientific publications.</li>
</ul>
<p>The development of <strong>VegX R package</strong> has been conducted in parallel to an extensive revision of Veg-X, which has led to the deployment of <strong>version 2.0</strong> of the exchange standard. Moreover, the package does not currently include all the main elements and sub-elements of Veg-X (see next section). This was a practical decision to enable a usable tool to be developed to meet the purpose described above and not be overly complex for users. Future versions of the package may allow more elements of the standard to be used while accounting for backward compatibility with Veg-X files conforming to version 2.0 or later versions.</p>
</div>
<div class="section level2">
<h2 id="data-structure-of-veg-x-ver--2-0">Data structure of Veg-X (ver. 2.0)<a class="anchor" aria-label="anchor" href="#data-structure-of-veg-x-ver--2-0"></a>
</h2>
<p>Veg-X has a non-hierarchical data structure. Different data elements relate to each other via identifiers in a flexible way. The following figure illustrates the relationships between the main elements of Veg-X.</p>
<div class="figure">
<img src="LogicalStructure.png" alt=""><p class="caption"><strong>Fig. 2</strong>. Main VegX elements and their logical relationships. Arrows indicate that an identifier of the origin element is referenced in the destination element. Accompanying numbers indicate the number of instances of the origin element that are allowed to be referenced in the destination element. Observations are in tinted boxes.</p>
</div>
<p>The following table provides the brief descriptions of the main elements of Veg-X (column ‘R’ indicates whether the sub-elements are currently implemented in the <strong>VegX R package</strong>):</p>
<table class="table">
<colgroup>
<col width="37%">
<col width="58%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Main element</th>
<th>Description</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>project</td>
<td>Describes the research context in which the dataset was created, including descriptions of over-all motivations and goals, funding, personnel, description of the study area etc.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>plot</td>
<td>A plot is a sampling location. It records the properties of the vegetation plot independent of time and can be referenced by many observations by links to the unique plot code.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>plotObservation</td>
<td>Observations made on a single plot and during a single date-time period. This allows all time dependent parameters to be grouped.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>organismIdentity</td>
<td>The identity of an organism occurring within the dataset. This is a name defined by the dataset author which may follow, or not, nomenclatural codes. It may be further related to a published name (taxon name, tcs:TaxonName) and or taxonomic name (taxon concept, tcs:TaxonConcept) in taxonDetermination.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>individualOrganismObservation</td>
<td>An observation applying to one occurrence of an organism (or part of an organism). It is a container for measurements made on the organism (e.g. diameter, height, crown dimensions, biomass, growth form, number of stems).</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>individualOrganism</td>
<td>An identified organism recorded during one or more individual organism observation events. Individuals may have an identification label (e.g. tree tag number).</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>aggregatedOrganismObservation</td>
<td>An observation applying to all occurrences of an organism based on an aggregation factor.</td>
<td></td>
</tr>
<tr class="even">
<td>stratumObservation</td>
<td>A specific observation applying to a stratum in a single plot during a single date-time period. Each stratum measurement may be referenced by observations of taxa within a plot. For example, abundance estimates of a taxa on a plot within a specific stratum.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>stratum</td>
<td>The specific definition of a stratum referred to by observations in the dataset. A stratum usually belongs to a ordered list that together are the set of strata definitions in use in a specific dataset.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>communityObservation</td>
<td>A container for measurements that apply to the entire plant community and made on a single plot during a single date-time period.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>siteObservation</td>
<td>A container for all the site (i.e. soil, climate, landuse, habitat, …) measurements made on a single plot during a single date-time period. Unlike other observation elements, it relates to <em>plotObservation</em> in a one-to-one relationship.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>surfaceType</td>
<td>The definition of a surface type, not the observation of cover on it.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>surfaceCoverObservation</td>
<td>A single cover measurement applying to a surface type in a single plot during a single date-time period.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>taxonDetermination</td>
<td>A specific relationship or assertion between two name concepts which are not part of the original definition of either of these concepts; possibly by a third party. This typically allows for an organism identity to be linked to a specific taxa treatment (taxon concept), according to a third party. Similar to a tcs TaxonRelationshipAssertion.</td>
<td>No</td>
</tr>
<tr class="odd">
<td>communityDetermination</td>
<td>An identification applying one or more community concepts to a plot observation by a party.</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationGrouping</td>
<td>An specific grouping of observation records, of any kind, that are grouped in the data management system owing to some common characteristic.</td>
<td>No</td>
</tr>
</tbody>
</table>
<p>Additional resource elements of the Veg-X schema are used to provide information about people, methods, bibliographic references, organism names and concepts:</p>
<table class="table">
<colgroup>
<col width="37%">
<col width="58%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Resource element</th>
<th>Description</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>party</td>
<td>Describes a responsible party (person or organization), and is typically used to name the originator of a resource or metadata document.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>attribute</td>
<td>A specific definition of a measured property. An attribute has to be one of three types: qualitative (unordered categorical variable, i.e. nominal), ordinal (ordered list of values) or quantitative (a numerical variable, either discrete or continuous.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>method</td>
<td>A specific method definition followed in the creation of the dataset (e.g. the measurement of pH, the estimation of plant abundance or the definition of strata).</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>protocol</td>
<td>A specific grouping of methods related by common action. A protocol may have many method or steps.</td>
<td>No</td>
</tr>
<tr class="odd">
<td>literatureCitation</td>
<td>Provides overview information about the literature, including citation string and DOI.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>organismName</td>
<td>The name of an organism used in the data set. This will normally be a nomenclatural unit of any rank (order, family, genus, species, subspecies, etc.). If it is a formal scientific name (not necessarily including authority) then the attribute ‘taxonName’ should be set to true. However, the organism name can be a morphospecies, a field name… cases in which the attribute ‘taxonName’ should be set to false.</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>taxonConcept</td>
<td>Representation of a taxon concept (i.e., an organism name and the organism description given by an author in a publication). A taxon concept may be referenced in an organism identity as the original concept used by the author of the data set, or it can be referenced in taxonDetermination allowing an organism identity to be mapped to a taxonomic concept by third parties after re-examination.</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>communityConcept</td>
<td>A name and some kind of definition of a community type, preferably a community name as used in a reference.</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level2">
<h2 id="details-of-the-main-veg-x-elements">Details of the main Veg-X elements<a class="anchor" aria-label="anchor" href="#details-of-the-main-veg-x-elements"></a>
</h2>
<p>In the following sections we detail the sub-elements of all <strong>main elements</strong> (except <em>communityDetermination</em> and <em>taxonDetermination</em>) in the <strong>Veg-X standard</strong>, along with some instructions on how the standard should be used. The <strong>VegX package</strong> maintains unique IDs for all main elements, for internal consistency. However, these IDs are largely hidden from the user (but you will see ID references in the descriptions below). From each element described, we indicate the combination of fields that uniquely identify them and are used to guide element merge. In the tables describing sub-elements, we use column ‘<strong>#</strong>’ to indicate the number of times they can (or must) occur to have a valid Veg-X document (‘1’ means they occur one and are required; ‘0..1’ means they are optional and can occur only once; ‘0..n’ means they are optional and can occur many times; ‘1..n’ means they must occur at least once, but can occur many times). As before, column ‘<strong>R</strong>’ (‘Yes’/‘No’/‘Partial’) indicates whether the sub-elements are currently implemented in the <strong>VegX R package</strong> (‘Partial’ indicates an implementation that is not complete).</p>
<div class="section level3">
<h3 id="element-project">Element <em>project</em><a class="anchor" aria-label="anchor" href="#element-project"></a>
</h3>
<p>A <em>project</em> element describes the research context in which the dataset was created, including descriptions of over-all motivations and goals, funding, personnel, description of the study area etc. The definition of Veg-X <em>project</em> elements was borrowed from the Ecological Metadata Language (v. 2.0.1). from the user’s perspective projects are uniquely identified by their sub-element <em>title</em>.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>title</td>
<td>Title of the project.</td>
<td>1..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>personnel</td>
<td>Contact and role information for people involved in the research project.</td>
<td>1..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>abstract</td>
<td>A brief description of the aims and findings of the project.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>funding</td>
<td>Funding information.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>studyAreaDescription</td>
<td>Description of the physical area associated with the research project, potentially including coverage, climate, geology, distrubances, etc.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>designDescription</td>
<td>Description of the design of the research project (specially overall plot placement).</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>relatedProjectID</td>
<td>A link to another project, by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>documentCitationID</td>
<td>A link, by ID, to the citation of a document describing the project.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-plot">Element <em>plot</em><a class="anchor" aria-label="anchor" href="#element-plot"></a>
</h3>
<p>A <em>plot</em> is a sampling location, represented as one or more points, lines, polygons, or volumes, and is the basis for experimentation or measurement. Its properties are assumed to be constant over time. A point within the plot may be used as center for relative coordinates, which are required to be Cartesian. Plots may have no explicit bounds, and may refer to an area of inference. A plot may be related to other plots in order to express parent-child, contiguity, or other type of relationship.</p>
<p>The element <em>plot</em> records the properties of the vegetation plot that are independent of time. The <strong>Veg-X standard</strong> allows storing globally unique identifiers for <em>plot</em> elements, but the <strong>VegX package</strong> uses its own set of IDs for internal consistency. From the perspective of the provider of a single data set, plots can be uniquely identified by their sub-element <em>plotName</em>. However, when merging data sets <em>plotUniqueIdentifier</em> from different sources, helps distinguishing plots that may have the same name but come from different sampled areas. Even though the standard allows different kinds of spatial relationships between plots to be specified, currently the R package only enables parent-child relationships to be specified.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotName</td>
<td>Name or label for a plot, unique within the data set</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>plotUniqueIdentifier</td>
<td>Plot identifier that is unique across the dataset, derived from the data source, and preferably globally unique.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>relatedPlot</td>
<td>A plot may be related spatially to other plots in order to express parent-child, sub-plot or contiguity.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>placementMethod</td>
<td>Strategy followed when placing this particular plot. Useful for example if different sampling strategies have been followed within one project.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>placementPartyID</td>
<td>A link to a party that participated in the establishment of the plot, by ID.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>placementNote</td>
<td>Additional comments or explanations regarding plot placement.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>location</td>
<td>Information regarding the location of the plot on earth’s surface.</td>
<td>0..1</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>geometry</td>
<td>Information regarding the geometry of the plot (area, shape, dimensions, coordinates, …) as well as the point within the plot that serves as the plot origin for location.</td>
<td>0..1</td>
<td>Partial</td>
</tr>
<tr class="odd">
<td>topography</td>
<td>Information regarding the shape and features of the surface on which the plot was placed (e.g. aspect, slope, …).</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>parentMaterial</td>
<td>Underlying geological material (generally bedrock or a superficial or drift deposit) in which soil horizons form.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
<p>Sub-elements <em>location</em> and <em>geometry</em> are specially important, as they contain the description of plot location and shape, respectively. For this reason we describe these child elements in some detail.</p>
<div class="section level4">
<h4 id="sub-element-location">Sub-element <em>location</em><a class="anchor" aria-label="anchor" href="#sub-element-location"></a>
</h4>
<p><em>Location</em> stores information regarding the location of a plot on the earth surface. Child elements <em>horizontalCoordinates</em> are used to store x-y coordinates in a spatial reference system. To avoid ambiguity there should be only one coordinate pair for a plot, and the implementation of the <strong>VegX R package</strong> follows this rule. However, the Veg-X schema can accomodate multiple coordinate measurements made by different parties or at different times. The same applies to <em>verticalCoordinates</em>, <em>locationInWaterBody</em> and <em>gridPosition</em> (the later two not covered in the R package).</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>horizontalCoordinates</td>
<td>Horizontal coordinates of a plot on the Earth’s surface (i.e. x-y coordinates in a spatial reference system).</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>verticalCoordinates</td>
<td>Elevation of the plot in respect to some vertical datum (such as the mean sea level or an elliposid).</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="odd">
<td>markers</td>
<td>Information about markers (like magnetic markers or wooden pegs) that help locating the plot. There should also be a description about where in the plot the markers are found (for example in the corners or in the centre).</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>locationInWaterBody</td>
<td>Location in respect to a water surface or shoreline.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>gridPosition</td>
<td>Position in a grid such as used for floristic surveys.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>authorLocation</td>
<td>Descriptive note about the original location described by author.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>locationNarrative</td>
<td>Text description that provides information useful for plot relocation.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>places</td>
<td>A collection of named places or geographic regions. Includes elements to indicate what type of place and which place/geo-region schema it was from.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="sub-element-geometry">Sub-element <em>geometry</em><a class="anchor" aria-label="anchor" href="#sub-element-geometry"></a>
</h4>
<p>Sub-element <em>geometry</em> stores information regarding the geometry of the plot (area, shape, dimensions, coordinates, …) as well as the point within the plot that serves as plot origin for location. Sub-element <em>geometry</em> may be lacking in plot-less vegetation observations. When present, the user of the schema has to choose one plot shape: <em>circle</em>, <em>rectangle</em>, <em>line</em> or <em>polygon</em>. Currently, the VegX R package allows storing information about area, shape and dimensions, but not plot origin, orientation and path.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>area</td>
<td>Total area of the plot. Usually recorded in square meters.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>shape</td>
<td>Plot’s shape: linear, rectangle, polygon or circle.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>plotOrigin</td>
<td>Definition of the position of the plot origin within the plot (here usually “center”). This is referred to in the horizontalCoordinates element under locationInPlot. The actual coordinates go into the location - horizontalCoordinates element</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>radius</td>
<td>Define the radius of circular plots. Usually recorded in meters.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>width</td>
<td>Width of a regular rectangle. In case of a square plot, the width of both sides.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>length</td>
<td>Length (largest dimension) of a rectangle plot or length of a linear plot.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>orientation</td>
<td>Orientation of the main axis of the plot (e.g. in degrees from North). For quadrat plots the axis closer to the N-S axis should be given.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>bandWidth</td>
<td>Distance from the linear plot axis. This distance delimits the surface included for measurements.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>path</td>
<td>Set of points conforming the path in a linear plot (i.e. a transect)</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>outerBoundary</td>
<td>Absolute or relative coordinates defining the outline of a polygon.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>innerBoundary</td>
<td>Coordinates defining any inner boundary of a polygon</td>
<td>0..1</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level3">
<h3 id="element-plotobservation">Element <em>plotObservation</em><a class="anchor" aria-label="anchor" href="#element-plotobservation"></a>
</h3>
<p>An element <em>plotObservation</em> is used to group all observations made on a single plot and during a single date-time period. While the <strong>Veg-X standard</strong> allows globally unique identifiers for <em>plotObservation</em> elements to be stored, the <strong>VegX package</strong> uses its own set of IDs for internal consistency. From the user’s perspective, plot observations are uniquely identified by the plot’s name (and its unique identifier, if present) and <em>obsStartDate</em>.</p>
<p>While the schema allows multiple references to <em>project</em> elements, the VegX package only enables a single reference to a <em>project</em> element to be specified (sub-element <em>projectID</em>). Similarly, the package allows only one party to be specified, among those involved in the plot survey (sub-element <em>observationPartyID</em>).</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotID</td>
<td>A link to a specific plot by the plot’s ID .</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>obsStartDate</td>
<td>The start date of this specific observation of the plot. Recorded in ISO 8601 date format: yyyy-mm-dd.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>obsEndDate</td>
<td>The end date of this specific observation of the plot. Recorded in ISO 8601 date format: yyyy-mm-dd.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>plotObservationUniqueIdentifier</td>
<td>Plot observation identifier that is unique across the dataset, derived from the data source, and preferably globally unique.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>projectID</td>
<td>A link to a specific ‘project’ by ID.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>previousObservationID</td>
<td>A link to previous plot observations.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>communityObservationID</td>
<td>A link to a specific community observation by ID. Note that the relationship is one-to-one. Only one community observation (with potentially many measurements inside) is allowed for each plot observation.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>siteObservationID</td>
<td>A link to a specific site observation by ID. Note that the relationship is one-to-one. Only one site observation (with potentially many measurements inside) is allowed for each plot observation.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>previousObservationID</td>
<td>A link to a previous plot observation, by ID. Not normally necessary as observations can be ordered via obsStartDate.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationPartyID</td>
<td>A link to a party that participated in the observation of the plot, by ID.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="odd">
<td>license</td>
<td>License linked to this plot observation.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>taxonomicQuality</td>
<td>Subjective assessment of the taxonomic quality on the plot.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNarrative</td>
<td>Additional unstructured observations useful for understanding the ecological attributes and significance of the plot observations.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationConditions</td>
<td>Conditions at the time of observation.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>referencePublication</td>
<td>Reference to an original publication and additionally a table or section within a publication.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationGroupingID</td>
<td>A reference to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNote</td>
<td>Additional comments or explanations pertaining to the observation event.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-organismidentity">Element <em>organismIdentity</em><a class="anchor" aria-label="anchor" href="#element-organismidentity"></a>
</h3>
<p>The identity of an organism (or a set of organisms) occurring within the dataset. This is initially a name defined by the dataset author which may or not be following nomenclatural codes. The identity may be complemented with an scientific name (taxon name) accepted according to a specified nomenclatural authority, by using the sub-element <em>preferredTaxonNomenclature</em>. The taxonomic concept (taxon name + reference defining the concept) that the author of the observation had in mind when observing the organism (or that a third party assumes he had in mind) can also be specified in the sub-element <em>originalIdentificationConcept</em>. Subsequent re-evaluations of the taxon concept (e.g. after inspection of the herbarium voucher) by third parties should be specified using the main element <em>taxonDetermination</em> (which are not currently supported by the <strong>VegX package</strong>).</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>originalOrganismNameID</td>
<td>A link, by ID, to an organism name (e.g. normally a taxon name, but not necessarily including the authority, or even field names, morphospecies, …) that the author of the dataset originally used to refer to an organism observed within the plot. The taxon names used as label the organism identity should not contain spelling errors, but they may not be the accepted name according to current nomenclature codes.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>originalIdentificationPartyID</td>
<td>A link to a party involved in the original organism identification (normally the author of the data set), by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>originalIdentificationNote</td>
<td>Additional comments or explanations pertaining to the original identification of the organism.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>voucher</td>
<td>Herbarium accession number for any archived voucher specimens.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>originalIdentificationConcept</td>
<td>The taxon concept originally associated to the organism identity. This may have been specified by the author of the data set, or it may be asserted by a third party based on information such as date of observation or geographic location.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>preferredTaxonNomenclature</td>
<td>The interpretation of the nomenclature that should be applied to organism identity, made after the observation event by the author of the data set or a third party. The sub-element <em>preferredTaxonNameID</em> points to an organism name that is the accepted name according to the current nomenclature.</td>
<td>0..1</td>
<td>Partial</td>
</tr>
</tbody>
</table>
<p>When displaying organism observations, the VegX package uses a field called <strong>organismIdentityName</strong> to name organisms with the following rules:</p>
<ul>
<li>If <em>preferredTaxonNomenclature</em> has NOT been defined then <em>organismIdentityName</em> is equal to the <em>originalOrganismName</em> (via <em>originalOrganismNameID</em>).</li>
<li>If <em>preferredTaxonNomenclature</em> has been defined then <em>organismIdentityName</em> is equal to the <em>preferredTaxonName</em> (via <em>preferredTaxonNameID</em>).</li>
</ul>
<div class="section level4">
<h4 id="sub-element-originalidentificationconcept">Sub-element <em>originalIdentificationConcept</em><a class="anchor" aria-label="anchor" href="#sub-element-originalidentificationconcept"></a>
</h4>
<p>Stores the taxon concept originally associated to the organism identity. This may have been specified by the author of the data set, or it may be asserted by a third party based on information such as date of observation or geographic location.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>taxonConceptID</td>
<td>A link to the taxon concept stated by the author, or as asserted by a third party based on information such as the date of the observation, geographic location, etc.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>conceptAssertionDate</td>
<td>Date of the taxon concept assertion. Recorded in ISO 8601 date format: yyyy-mm-dd.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>conceptAssertionPartyID</td>
<td>A link, by ID, to a party involved in the assertion of the original taxon concept.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>conceptAssertionNote</td>
<td>Additional comments or explanations pertaining to the assertion of the original taxon concept.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="sub-element-preferredtaxonnomenclature">Sub-element <em>preferredTaxonNomenclature</em><a class="anchor" aria-label="anchor" href="#sub-element-preferredtaxonnomenclature"></a>
</h4>
<p>Stores the interpretation of the nomenclature that should be applied to organism identity, made after the observation event by the author of the data set or a third party.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>preferredTaxonNameID</td>
<td>A link to a scientific taxon name (i.e. an organism name whose attribute ‘taxonName’ is true) accepted to label the organism identity appropriately, as stated by the author of the data set or a third party responsible for its nomenclature.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>interpretationDate</td>
<td>Date for the last nomenclature revision applied to this organism identity. Recorded in ISO 8601 date format: yyyy-mm-dd.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>interpretationSource</td>
<td>A string describing the source for the last nomenclature interpretation applied to this organism identity (i.e. the Plant List).</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>interpretationCitationID</td>
<td>A link to the publication where nomenclature interpretation is described.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>interpretationPartyID</td>
<td>A link to a party who undertook the nomenclarure revision.</td>
<td>0..n</td>
<td>Partial</td>
</tr>
<tr class="even">
<td>interpretationSource</td>
<td>Additional comments or explanations pertaining to the nomenclature interpretation.</td>
<td>0..n</td>
<td>Np</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level3">
<h3 id="element-stratum">Element <em>stratum</em><a class="anchor" aria-label="anchor" href="#element-stratum"></a>
</h3>
<p>The definition of a stratum, not the observation of a stratum. An individual stratum usually belongs to a ordered list that together are the set of strata definitions in use within a specific dataset. This set of strata will normally have been defined according to the same method, but individual stratum may also be assigned a method. Strata that are defined from limits in a quantitative measurement, like height, the user of the Veg-X standard can use the method pointed by <em>methodID</em> to describe the quantitative attribute associated to the stratum definition (e.g. height in m). From the package user’s perspective, strata are uniquely identified by the name of the stratum definition method and the <em>stratumName</em>.</p>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>stratumName</td>
<td>Name associated with this stratum and which identifies it.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>methodID</td>
<td>A reference to a specific method used to define this stratum.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>definition</td>
<td>A longer description of the stratum definition.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>order</td>
<td>An indication of a position in an ordered sequence of strata.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>lowerLimit</td>
<td>Lower limit of the stratum in some known dimension (e.g. height) defined in the attribute of the method pointed to by ‘methodID’.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>upperLimit</td>
<td>Upper limit of the stratum in some known dimension (e.g. height) defined in the attribute of the method pointed to by ‘methodID’.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-stratumobservation">Element <em>stratumObservation</em><a class="anchor" aria-label="anchor" href="#element-stratumobservation"></a>
</h3>
<p>A <em>stratumObservation</em> is a specific observation applying to a stratum in a single plot during a single date-time period. Each stratum observation may be referenced by observations of taxa or individuals within a plot. For example, abundance estimates of a taxa on a plot within a specific stratum. In addition, the <em>stratumObservation</em> may contain measurements of the lower and upper vertical limits of the stratum (if those are not fixed by the stratum definition) and an assessment of plant abundance like cover or number of individuals. A <em>stratumObservation</em> always contains a reference to a <em>plotObservation</em>, where contextual information lies (plot, project, parties, date-time period). It also contains a reference to a <em>stratum</em>, which contains its definition. From the package user’s perspective, stratum observations are uniquely identified by the name of the stratum definition method, the stratum name and the plot observation.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>stratumID</td>
<td>A reference to a specific stratum by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>plotObservationID</td>
<td>A reference to a specific plotObservation.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>lowerLimitMeasurement</td>
<td>A measurement of the lower limit (i.e. height) of the stratum.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>upperLimitMeasurement</td>
<td>A measurement of the upper limit (i.e. height) of the stratum.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>stratumMeasurement</td>
<td>A measurement (e.g. plant cover, or individual count) made in the stratum.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>observationGroupingID</td>
<td>A reference to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNote</td>
<td>Additional comments or explanations regarding this observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-aggregateorganismobservation">Element <em>aggregateOrganismObservation</em><a class="anchor" aria-label="anchor" href="#element-aggregateorganismobservation"></a>
</h3>
<p>An <em>aggregateOrganismObservation</em> is an observation applying to all occurrences of an organism (e.g. a taxon). An <em>aggregateOrganismObservation</em> contains a reference to a single <em>plotObservation</em> and a link to a <em>organismIdentityID</em>, which can be linked to all the taxon identification information. Optionally, it may also link to a <em>stratumObservation</em>. It may contain one of several sub-elements <em>aggregateMeasurement</em>, each of them being an assessment of the overall occurrence of an organism in a Plot (e.g. number of stems, percentage cover, total biomass, basal area). If there is no instance of <em>aggregateMeasurement</em>, then the taxon is understood to be simply present. From package user’s perspective, aggregate organism observations are uniquely identified by plot observation and organism identity, and by the stratum observation when defined.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotObservationID</td>
<td>A link to a specific plot observation by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>organismIdentityID</td>
<td>A link to a specific organism identity by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>aggregateOrganismMeasurement</td>
<td>A measurement for a aggregate organism value (e.g. plant cover of a taxon). Values can be further defined uppervalue, accuracy etc. Many measurements (e.g. counts, cover, basal area…) can be added to the same aggregate organism observation.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>heightMeasurement</td>
<td>Optional height at which the aggregated observation was made, e.g. in meters. It applies to all aggregate measurements included in this aggregateOrganismObservation.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>stratumObservationID</td>
<td>A link to a specific stratumObservation by ID. It applies to all aggregate measurements included in this aggregateOrganismObservation.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>observationGroupingID</td>
<td>A link to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNote</td>
<td>Additional comments or explanations regarding this observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-individualorganism">Element <em>individualOrganism</em><a class="anchor" aria-label="anchor" href="#element-individualorganism"></a>
</h3>
<p>An element <em>individualOrganism</em> represents an organism recorded during one or more observation events and identified through an identification label (e.g. tree tag number). In Veg-X documents, individual organisms may or may not have been given a taxon name (i.e. a link via sub-element <em>organismIdentityID</em>), and the standard allows specifying the relative position of individuals within the plot to which they belong, as well as to specify related individuals. From the perspective of the <strong>VegX package</strong> user, individual organisms are identified by plot identity (i.e. name and unique identifier if defined) and the label of the individual organism.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotID</td>
<td>A reference to a specific plot by the plot ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>individualOrganismLabel</td>
<td>A label that is associated with an individual (e.g. a numerical tree tag).</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>organismIdentityID</td>
<td>A reference to a specific organismIdentity by ID.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>birthDate</td>
<td>Date of birth recorded in ISO 8601 date format: yyyy-mm-dd.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>relatedIndividual</td>
<td>An item may be related or connected in some way to other items. For example fused stems or epiphytic relationships.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>location</td>
<td>Information regarding the location of an organism on the earth’s surface, either absolute or relative to the plot origin or to a related individual.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>individualOrganismNote</td>
<td>For specifying additional comments or explanations pertaining to the individual organism.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
<p>While <em>relatedIndividual</em> is used to link to other organisms in an arbitrary relationship (e.g. epiphytes or fused stems), quantitative spatial relationships should be specified in <em>location</em>, as discussed below.</p>
<div class="section level4">
<h4 id="sub-element-location-1">Sub-element <em>location</em><a class="anchor" aria-label="anchor" href="#sub-element-location-1"></a>
</h4>
<p>The <em>location</em> of an organism is assumed to be constant in time during the organism lifespan. Sub-elements <em>horizontalLocation</em> are used to store x-y (or polar) coordinates of organism, either absolute or in relation to the plot origin or a related organism. To avoid ambiguity there should be only one coordinate pair for an organism. However, the Veg-X schema can accomodate multiple measurements made by different parties or at different times. The R package does not yet support <em>location</em> elements for organisms.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>horizontalLocation</td>
<td>Horizontal location of an organism on the Earth’s surface (absolute or relative to a reference point such as the plot origin or a related individual).</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>verticalLocation</td>
<td>Elevation of the item in respect to some vertical datum (such as the mean sea level or an ellipsoid) or in relation to the plot origin or to a relatedIndividual.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>markers</td>
<td>Information about markers (like tags) that help locating the organism.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>quadrant</td>
<td>One out of four quadrats such as those used in Point-Centred Quarter Method (PCQM)</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level3">
<h3 id="element-individualorganismobservation">Element <em>individualOrganismObservation</em><a class="anchor" aria-label="anchor" href="#element-individualorganismobservation"></a>
</h3>
<p>An element <em>individualOrganismObservation</em> is an observation applying to one occurrence of an organism (or part of an organism). It is a container for measurements made on the organism (e.g. diameter, height, crown dimensions, biomass, growth form, number of stems). An <em>individualOrganismObservation</em> contains a reference to a unique <em>plotObservation</em> and to an <em>individualOrganism</em>. Optionally, the <em>individualOrganismObservation</em> may link to a <em>stratumObservation</em>. Regarding measurements, the schema includes specific subelements to store the measurement of plant height and stem diameter, the latter including or not a measurement of distance from the ground at which diameter was measured. Other measurements such as growth form, canopy dimensions, health status… can be included in sub-elements <em>individualOrganismMeasurement</em> (for simple measurements) or <em>individualOrganismMultipleMeasurement</em> for multiple (tuple) measurements.</p>
<p>From the package user’s perspective, individual organism observations are uniquely identified by plot observation and the label of the individual organism.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotObservationID</td>
<td>A reference to a specific plotObservation by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>individualOrganismID</td>
<td>A reference to a specific individualOrganism by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>stratumObservationID</td>
<td>A reference to a specific stratum observation that this individual was measured in on the plot.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>heightMeasurement</td>
<td>Measurement of the maximum height reached by the observed individual.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>diameterMeasurement</td>
<td>Diameter of the stem without explicit measurement of base distance (may be defined in the measurement method definition)</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>diameterBaseDistanceMeasurement</td>
<td>A container for diameter measurements at a given distance along the stem from the ground.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>individualOrganismMeasurement</td>
<td>A measurement applying to the observed individual. This includes qualitative, ordinal or quantitative assessments of form, health, dimensions of components, …</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>individualOrganismMultipleMeasurement</td>
<td>A n-tuple of a related measurements (e.g. paired data such as length, width and height to calculate volume). The definition of the relationship type is left open to the user, but is intended to allow specifying a set of measurements that have to be considered together.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationGroupingID</td>
<td>A reference to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationNote</td>
<td>Additional comments or explanations regarding this observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-communityobservation">Element <em>communityObservation</em><a class="anchor" aria-label="anchor" href="#element-communityobservation"></a>
</h3>
<p>A <em>communityObservation</em> is a container for all measurements that apply to the entire plant community and are made on a single plot during a single date-time period. Unlike other observation elements, it relates to <em>plotObservation</em> in a one-to-one relationship. The reason is because, unlike other entities, there is a single entity to which measurements refer. There is no specific variable to identify uniquely community observations, but they are uniquely identified by their related plot observation.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotObservationID</td>
<td>A reference to a specific plotObservation.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>communityMeasurement</td>
<td>A measurement (e.g. number of individuals, basal area) applying to the whole plant community or forest stand.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>successionalType</td>
<td>Description of the assumed successional status of the plot. This description is of necessity highly subjective.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationGroupingID</td>
<td>A reference to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNote</td>
<td>Additional comments or explanations regarding this observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-siteobservation">Element <em>siteObservation</em><a class="anchor" aria-label="anchor" href="#element-siteobservation"></a>
</h3>
<p>A <em>siteObservation</em> is a container for all the site (i.e. soil, climate, landuse, habitat, …) measurements made on a single plot during a single date-time period. Unlike other observation elements, it relates to <em>plotObservation</em> in a one-to-one relationship. The reason is because, unlike other entities, there is a single entity (i.e. the site) to which all measurements refer. There is no specific variable to identify uniquely site observations, but they are uniquely identified by their related plot observation. Abiotic (i.e. soil, climate or water body) measurements are a special kind of measurements within the Veg-X schema, in the sense that: (1) they can have IDs and can thus be related to each other: (2) and they can have relative coordinates of the measurement within the plot, in the same way as individuals.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotObservationID</td>
<td>A reference to a specific plotObservation.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>soilMeasurement</td>
<td>A measurement of a soil attribute (soil chemistry, soil texture, structure, …).</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>climateMeasurement</td>
<td>A measurement of a climate attribute.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>waterBodyMeasurement</td>
<td>A measurement of an attributes of a water body within the plot (e.g. water level, not for soil water, which should be included in soilMeasurement).</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>soilType</td>
<td>A specific soil type, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>humusType</td>
<td>A specific humus type, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>climateType</td>
<td>A specific climate type, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>hydrologicRegimeType</td>
<td>Reflection of frequency and duration of water level variations, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>legalProtection</td>
<td>Legal protection status of the plot during the plot observation. Recommended that this is from a closed list of legal protection status types.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>landuse</td>
<td>A specific land use type, for example pasture, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>habitat</td>
<td>A specific habitat type, applying to this plot during the plot observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="even">
<td>observationGroupingID</td>
<td>A reference to a specific observation grouping by ID.</td>
<td>0..n</td>
<td>No</td>
</tr>
<tr class="odd">
<td>observationNote</td>
<td>Additional comments or explanations regarding this observation.</td>
<td>0..n</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-surfacetypes">Element <em>surfaceTypes</em><a class="anchor" aria-label="anchor" href="#element-surfacetypes"></a>
</h3>
<p>The definition of surface types, not the observation of cover on them. From the package user’s perspective, surface types are uniquely identified by the name of their definition method the surface type name.</p>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>surfaceName</td>
<td>Name associated with this surface type and which identifies it.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>methodID</td>
<td>A reference to a specific method used to define this surface type.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>definition</td>
<td>A longer description of the surface type definition.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-surfacecoverobservation">Element <em>surfaceCoverObservation</em><a class="anchor" aria-label="anchor" href="#element-surfacecoverobservation"></a>
</h3>
<p>A <em>surfaceCoverObservation</em> is a single cover measurement applying to a surface type in a single plot during a single date-time period. From the package user’s perspective, surface cover observations are uniquely identified by surface type and plot observation.</p>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>plotObservationID</td>
<td>A link to a specific plotObservation by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>surfaceTypeID</td>
<td>A link to a specific surface type by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>coverMeasurement</td>
<td>The cover measurement, usually in percent cover of the surface when projected to the ground.</td>
<td>1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-observationgrouping">Element <em>observationGrouping</em><a class="anchor" aria-label="anchor" href="#element-observationgrouping"></a>
</h3>
<p>A specific grouping of observation records, of any kind, that are grouped in the data management system owing to some common characteristic. For example, records that represent revisits to the same area for monitoring purposes can be linked together through this entity. Note that some specific groupings are already defined in the schema and therefore they should not be repeated (e.g. the grouping of observations made on a specific plot during a specific time is a plotObservation).</p>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>name</td>
<td>The unique name of a specific grouping entity that is subsequently referenced by specific observations.</td>
<td>1</td>
<td>No</td>
</tr>
<tr class="even">
<td>type</td>
<td>The grouping entity type. For example, grouping of individual organism observations for the purposes of describing a physical relationship. Recommended that a closed-list is developed and used.</td>
<td>1</td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level2">
<h2 id="details-of-resource-elements">Details of resource elements<a class="anchor" aria-label="anchor" href="#details-of-resource-elements"></a>
</h2>
<p>In the following sections we detail the sub-elements of all <strong>resource elements</strong> (except <em>protocol</em> and <em>communityConcept</em>) in the <strong>Veg-X standard</strong>. From each element described, we indicate the combination of fields that uniquely identify them and are used to guide element merge.</p>
<div class="section level3">
<h3 id="element-party">Element <em>party</em><a class="anchor" aria-label="anchor" href="#element-party"></a>
</h3>
<p>A <em>party</em> element describes a responsible party (person, organization or a position), and is typically used to name the originator of a resource or metadata document. Parties are uniquely identified by the party name, which is either an <em>individualName</em>, <em>organizationName</em> or <em>positionName</em>.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>individualName</td>
<td>The full name of the person being described.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>organizationName</td>
<td>The full name of the organization being described.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>positionName</td>
<td>The name of the title or position associated with the resource.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>address</td>
<td>The full address information for a given responsible party entry.</td>
<td>1..n</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>phone</td>
<td>Information about the contact’s telephone.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>electronicMailAddress</td>
<td>The email address of the contact.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>onlineURL</td>
<td>A link to associated online information, usually a web site.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-literaturecitation">Element <em>literatureCitation</em><a class="anchor" aria-label="anchor" href="#element-literaturecitation"></a>
</h3>
<p>An element <em>literatureCitation</em> provides information about a literature reference, including citation string and DOI.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>citationString</td>
<td>A string indicating the citation reference</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>citationDOI</td>
<td>A string indicating the DOI that points to the resource.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-method">Element <em>method</em><a class="anchor" aria-label="anchor" href="#element-method"></a>
</h3>
<p>An element <em>method</em> provides the definition of a specific method followed in the creation of the dataset (e.g. the measurement of pH, the estimation of plant abundance or the definition of strata). From the perspective of the <strong>VegX package</strong> user, methods are identified uniquely through their element <em>name</em>. An important sub-element of a <em>method</em> is its <em>subject</em>, which contains the description of an attribute class to which the method applies, and is used for combining values which may be initially obtained using different methods. For example, subject would be pH measurement of upper soil solution, whereas a particular methods for this subject would be the measurement in water or measurement in 0.01 mol CaCl. All attributes pointing to a given <em>method</em> are assumed to apply to the same <em>subject</em>.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>name</td>
<td>Name associated with the method. For example, “percent cover”.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>description</td>
<td>A brief description of the method (e.g., measured parameter or basal area of all stems &gt; 10 cm dbh or counts of all saplings &gt;1.35 m tall and less than 2 cm dbh).</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>subject</td>
<td>The description of an attribute class for comparative purposes. If two methods measure the same attribute (e.g. plant cover), but with different degrees of precision and accuracy, setting ‘subject’ to ‘plant cover’ allows combining their values. All attributes pointing to the same method are assumed to apply to the same subject.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>protocolID</td>
<td>A reference to a specific protocol by its ID.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="odd">
<td>citationID</td>
<td>A reference to a specific citation of literature, by ID, where the method is explained in length.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-attribute">Element <em>attribute</em><a class="anchor" aria-label="anchor" href="#element-attribute"></a>
</h3>
<p>An <em>attribute</em> element contains the definition of a specific measured property. An attribute has to be one of three types: <em>qualitative</em> (unordered categorical variable, i.e. nominal), <em>ordinal</em> (ordered list of values) or <em>quantitative</em> (a numerical variable, either discrete or continuous. The sub-elements of attributes depend on its type.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (qualitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>methodID</td>
<td>A reference to a specific method that describes the context for the qualitative code.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>code</td>
<td>The label of the category used for measurement values.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>definition</td>
<td>Longer description of the definition of the category.</td>
<td>0..1</td>
<td>No</td>
</tr>
</tbody>
</table>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (ordinal)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>methodID</td>
<td>A reference to a method that describes the context for the ordinal code.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>code</td>
<td>Ordinal class code (e.g. a value like “+” or “1” in Braun-Blanquet cover scale)</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>definition</td>
<td>Longer description of the definition of the ordinal class. For example, “&gt;1-5 % percent cover” for code “1” in an ordinal cover scale.</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>lowerLimit</td>
<td>Lower limit of the ordinal class in an associated quantitative scale (e.g. 10% cover in a cover class)</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>upperLimit</td>
<td>Upper limit of the ordinal class in an associated quantitative scale (e.g. 25% cover in a cover class)</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>order</td>
<td>Explicit order in the sequence of ordinal values to which this class belongs.</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>methodID</td>
<td>A reference to a specific method that describes the context for the quantitative attribute.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>unit</td>
<td>Unit of measurement (e.g. mm, cm, square meters, number of individuals).</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>precision</td>
<td>The smallest place value to which the measurement is expressed (eg, if pi is represented as 3.14, then its precision is .01).</td>
<td>0..1</td>
<td>No</td>
</tr>
<tr class="even">
<td>lowerLimit</td>
<td>Potential lower limit of the measurement</td>
<td>0..1</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>upperLimit</td>
<td>Potential upper limit of the measurement</td>
<td>0..1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-organismname">Element <em>organismName</em><a class="anchor" aria-label="anchor" href="#element-organismname"></a>
</h3>
<p>An element <em>organismName</em> is simply a string with the name an organism used in the data set. This will normally be a nomenclatural unit of any rank (order, family, genus, species, subspecies, etc.). If it is a formal scientific name (not necessarily including authority) then the attribute ‘taxonName’ should be set to true. However, the organism name can be a morphospecies, a field name… cases in which the attribute ‘taxonName’ should be set to false.</p>
<table style="width:100%;" class="table">
<colgroup>
<col width="30%">
<col width="61%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Attribute</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody><tr class="odd">
<td>taxonName</td>
<td>A flag to identify that the organism name is a taxon name (i.e. a name according to a nomenclature code)</td>
<td>1</td>
<td>Yes</td>
</tr></tbody>
</table>
</div>
<div class="section level3">
<h3 id="element-taxonconcept">Element <em>taxonConcept</em><a class="anchor" aria-label="anchor" href="#element-taxonconcept"></a>
</h3>
<p>The representation of a taxon concept (i.e., an organism name and the organism description given by an author in a publication). A taxon concept may be referenced in an organism identity as the original concept used by the author of the data set, or it can be referenced in taxonDetermination allowing an organism identity to be mapped to a taxonomic concept by third parties after re-examination. Taxon concepts are uniquely determined by the organism (normally a taxon) name and a bibliographic citation.</p>
<table class="table">
<colgroup>
<col width="31%">
<col width="60%">
<col width="3%">
<col width="3%">
</colgroup>
<thead><tr class="header">
<th>Sub-element (quantitative)</th>
<th>Description</th>
<th>#</th>
<th>R</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>organismNameID</td>
<td>A link to a specific organism name by ID.</td>
<td>1</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>accordingToCitationID</td>
<td>A link to a bibliographic reference by ID where the taxon concept is described.</td>
<td>1</td>
<td>Yes</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<ul>
<li>Franz NM &amp; Peet RK (2009). Towards a language for mapping relationships among taxonomic concepts. <em>Systematics and Biodiversity</em> 7: 5-20.</li>
<li>Jones MB, Schildhauer MP, Reichman OJ &amp; Bowers S (2006). The new bioinformatics: integrating ecological data from the gene to the biosphere. <em>Annual Review of Ecology, Evolution and Systematics</em> 37: 519-544.</li>
<li>Wiser SK, Spencer N, De Caceres M, Kleikamp M, Boyle B &amp; Peet RK (2011). Veg-X - an exchange standard for plot-based vegetation data. <em>Journal of Vegetation Science</em> 22: 598-609.</li>
</ul>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Miquel De Cáceres, Sebastian Schmidtlein, Christian König.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
